verticaltabs.js

```diff
@@ -617,6 +636,40 @@ VerticalTabs.prototype = {
     }
   },
 
+  resizeTabs: function () {
```

Comment:
`resizeTabs` and `actuallyResizeTabs` is confusing. Could we rename `resizeTabs` to something else?